test/integration/noderesourcetopology_cache_test.go

```diff
@@ -264,6 +268,109 @@ func TestTopologyCachePluginWithoutUpdates(t *testing.T) {
 						}).Obj(),
 			},
 		},
+		{
+			name: "GU pod: DiscardReservedNodes: allows scheduling on both Zones",
```

Comment:
IIUC this test wants to demonstrate that DiscardReservedNodes can use both NUMA zones of a node without needing a NRT object update, comparing with OverReserve which cannot (because it overallocates).

So don't exercise yet the fact that the DiscardedReservedNodes is *preventing* access to a node when it is supposed to do, right?

Reply:
Yes, it's hard to check `preventing` access behaviour, cause Scheduler will retry scheduling when the node is locked and it will finally be successful.

Reply:
ok, and there's no way right (besides awkwardly parsing the logs) now to even verify the retry count.
I think really need to start the observability conversation.